File: src/test/java/com/gamebros/purepazaak/entity/FlipCardTest.java
package com.gamebros.purepazaak;

import com.gamebros.purepazaak.entity.FlipCard;
import org.junit.Test;
import static org.junit.Assert.assertEquals;

public class FlipCardTest {
  @Test
  public void testFlip() {
    FlipCard subject = new FlipCard(6);

    subject.flip();
    assertEquals(-6, subject.getValue());

    subject.flip();
    assertEquals(6, subject.getValue());
  }
}

File: core/src/test/kotlin/com/github/nomad145/purepazaak/core/card/FlipCardTest.kt
package com.github.nomad145.purepazaak.core.card

import org.junit.Assert.*
import org.junit.Test

class FlipCardTest{
    @Test
    fun testValueGetter(){
        val test = FlipCard(2)

        assertEquals(2,test.value)
        assertEquals("cards/flip.png", test.getImagePath())
    }
}

